### ISSUE TITLE
Nvidia drivers on 21.10

### ISSUE THREAD

**Issue by Kav57** [EtienneMueller/razer-blade-data-science#3]
Hello, 

Thank you so much for your guide, its really great to see that it is possible to get the razer blade 15 to work with ubuntu! 

I am currently trying to set up a dual boot on my razer blade 15 advanced but I keep running into issues with the nvidia drivers. I have tried running 20.04 and I'm currently running a live usb boot of 21.10 but I keep having issues with the nvidia settings. I have tried installing a number of different drivers but every time I run "nvidia-smi" I receive an error saying
NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver. Make sure that the latest NVIDIA driver is installed and running.

I was wondering if you encountered any issues like this at all and had any advice? I have  searched a lot of forums but nothing has worked so far 

Thanks again!
James

**Comment by EtienneMueller** [comment]
Hi James,

sorry for the incredibly late reply. I haven't had a Blade for a while and didn't update much in this repo. But I got a new one and will update everything accordingly.

Although I suppose that you resolved your problem by now: I had the same problem as you described. After the installation `nvidia-smi` could nt communicate with the driver. I had to go into *Software & Updated* and activate the Nvidia drivers again. Then run `sudo ubuntu-drivers install` in the terminal and reboot and everything worked fine again.

Hope that helps.

Cheers,
Etienne

**Comment by EtienneMueller** [comment]
Hi James,

sorry for the incredibly late reply. I haven't had a Blade for a while and didn't update much in this repo. But I got a new one and will update everything accordingly.

Although I suppose that you resolved your problem by now: I had the same problem as you described. After the installation `nvidia-smi` could nt communicate with the driver. I had to go into *Software & Updated* and activate the Nvidia drivers again. Then run `sudo ubuntu-drivers install` in the terminal and reboot and everything worked fine again.

Hope that helps.

Cheers,
Etienne
